PULL_REQUEST_TEMPLATE

```diff
@@ -5,33 +5,33 @@ Please include a summary of the change and which issue is fixed. Please also inc
 List any dependencies that are required for this change.
 -->
 
+This pull request fixes issue #<number>
+
 ## Submitter Checklist:
 
 - [ ] Submitted a [ticket](https://github.com/brave/brave-ios/issues) for my issue if one did not already exist.
```

Comment:
Is this needed if we have `This pull request fixes issue #<number>` few lines above

Reply:
Yeah maybe not 🤔